app/components/Form/ProjectContactFormSummary.tsx

```diff
@@ -104,44 +102,36 @@ const ProjectContactFormSummary: React.FC<Props> = (props) => {
   return (
     <>
       <h3>Project Contacts</h3>
-
-      {areContactsEmpty ? (
-        <p>
-          <em>Project contacts not added</em>
-        </p>
-      ) : (
-        <>
-          <label>Primary Contact</label>
-          {!primaryContact ? (
-            <dd>
-              <em>Not added</em>
-            </dd>
-          ) : (
-            <FormBase
-              key={primaryContact?.node.id}
-              tagName={"dl"}
-              theme={readOnlyTheme}
-              schema={projectContactSchema as JSONSchema7}
-              uiSchema={createProjectContactUiSchema(
-                primaryContact ? primaryContact.node.fullName : ""
-              )}
-              formData={primaryContact.node}
-              formContext={{
-                query: props.query,
-                form: primaryContact.node,
-              }}
-            />
-          )}
-          {<label>Secondary Contacts</label>}
-          {areSecondaryContactsEmpty ? (
-            <dd>
-              <em>Not added</em>
-            </dd>
-          ) : (
-            contactsJSX
+      <>
+        <label>Primary Contact</label>
+        <FormBase
+          liveValidate
+          key={primaryContact?.node.id}
+          tagName={"dl"}
+          theme={readOnlyTheme}
+          schema={projectContactSchema as JSONSchema7}
+          uiSchema={createProjectContactUiSchema(
+            primaryContact ? (
+              primaryContact?.node.fullName
+            ) : (
+              <em>Primary contact not added</em>
+            )
           )}
-        </>
-      )}
+          formData={primaryContactNode?.node.newFormData}
+          formContext={{
+            query: props.query,
+            form: primaryContact?.node,
+          }}
+        />
+        {<label>Secondary Contacts</label>}
+        {secondaryContacts.length === 0 ? (
+          <dd>
+            <em>Secondary contact not added</em>
+          </dd>
```

Comment:
```suggestion
          <em>Secondary contact not added</em>
```
I think the `dd` is what is causing the accessibility issue in the e2e tests

Reply:
I've opened a new issue: #563 this effects multiple forms.